python-challenges/automated-file-organizer-v2/devashmit/organizer.py

```
@@ -0,0 +1,54 @@
+    "code":      {".py",".js",".ts",".html",".css",".json",".xml",".yaml",".sh",".jsx",".tsx"},
+}
+
+def get_category(ext: str) -> str:
+    for cat, exts in CATEGORIES.items():
+        if ext.lower() in exts: return cat
+    return "others"
+
+def organize(source: Path, dry_run: bool) -> None:
+    moved = 0
+    for item in source.iterdir():
+        if item.is_dir() or item.name.startswith(".") or item.name == "organizer.log": continue
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)